Shared/ShortTimeSpan.cs

```diff
@@ -58,6 +62,32 @@ public static ShortTimeSpan FromSeconds(double seconds)
             return new ShortTimeSpan { unit = unit, Value = Math.Round(s, 3) };
         }
 
+        /// <summary>
+        /// For formating of timespans longer than 1 minute like "1 d 1 h 14m" instead of "90840 s"
+        /// </summary>
+        /// <param name="timeSpan"></param>
+        /// <returns></returns>
+        public static string LongTimeSpanFormat(TimeSpan timeSpan)
+        {
+            string result = "";
+            if (timeSpan.TotalSeconds == 0)
```

Comment:
This should probably be timeSpan.TotalSeconds < 1. Since TotalSeconds is a double.